src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.ts

```diff
@@ -67,6 +67,7 @@ export class OsdListComponent extends ListWithDetails implements OnInit {
 
   selection = new CdTableSelection();
   osds: any[] = [];
+  device_count: any;
```

Comment:
Not needed when using the suggested `get_osd_device_count` function. Remove it please.
```suggestion
```